gm-data-geonames/src/main/resources/reference.conf

```diff
@@ -0,0 +1,56 @@
+geomesa = {
+	sfts = {
+		geonames = {
+			type-name = "geonames"
+			attributes = [
+				{ name = "geonameid", type = "Integer", index = false },
+				{ name = "name", type = "String", index = true },
+				{ name = "asciiname", type = "String", index = false },
+				{ name = "alternatenames", type = "List[String]", index = true },
+				{ name = "featureclass", type = "String", index = false },
+				{ name = "featurecode", type = "String", index = false },
+				{ name = "countrycode", type = "String", index = false },
+				{ name = "cc2", type = "String", index = false },
+				{ name = "admin1code", type = "String", index = false },
+				{ name = "admin2code", type = "String", index = false },
+				{ name = "admin3code", type = "String", index = false },
+				{ name = "admin4code", type = "String", index = false },
+				{ name = "population", type = "Long", index = false },
+				{ name = "elevation", type = "Integer", index = false },
+				{ name = "dem", type = "Integer", index = false },
+				{ name = "timezone", type = "String", index = false },
+				{ name = "modificationdate", type = "Date", index = false },
+				{ name = "geom", type = "Point", index = true, srid = 4326, default = true }
+			]
+		}
+	},
+	converters = {
+		geonames = {
+			type = "delimited-text",
+			format = "MYSQL",
```

Comment:
[picky] add small comment on this line indicating that the data is tsv but in MYSQL format